common/persistence/cassandra/cassandraPersistence.go

```diff
@@ -731,6 +731,15 @@ workflow_state = ? ` +
 		`and task_id > ? ` +
 		`and task_id <= ?`
 
+	templateGetDLQReplicationTaskQuery = `SELECT replication ` +
+		`FROM executions ` +
+		`WHERE shard_id = ? ` +
+		`and type = ? ` +
+		`and domain_id = ? ` +
+		`and workflow_id = ? ` +
+		`and run_id = ? ` +
+		`limit 1`
```

Comment:
As discussed, If possible we would prefer to show the real backlog for easier view of num of messages in DLQ.

Reply:
Updated